http/http-binding/src/main/java/software/amazon/smithy/java/http/binding/HttpQuerySerializer.java

```diff
@@ -134,4 +134,75 @@ public void writeTimestamp(Schema schema, Instant value) {
             writeQuery(queryTrait, formatter.writeString(value));
         }
     }
+
+    private class ListElementSerializer extends SpecificShapeSerializer {
+        private final HttpQueryTrait parentTrait;
+        ListElementSerializer(HttpQueryTrait trait) {
+            this.parentTrait = trait;
+        }
+
+        @Override
+        public void writeBoolean(Schema schema, boolean value) {
+            writeQuery(parentTrait, Boolean.toString(value));
+        }
+
+        @Override
+        public void writeShort(Schema schema, short value) {
+            writeQuery(parentTrait, Short.toString(value));
+        }
+
+        @Override
+        public void writeByte(Schema schema, byte value) {
+            writeQuery(parentTrait, Byte.toString(value));
+        }
+
+        @Override
+        public void writeInteger(Schema schema, int value) {
+            writeQuery(parentTrait, Integer.toString(value));
+        }
+
+        @Override
+        public void writeLong(Schema schema, long value) {
+            writeQuery(parentTrait, Long.toString(value));
+        }
+
+        @Override
+        public void writeFloat(Schema schema, float value) {
+            writeQuery(parentTrait, Float.toString(value));
+        }
+
+        @Override
+        public void writeDouble(Schema schema, double value) {
+            writeQuery(parentTrait, Double.toString(value));
+        }
+
+        @Override
+        public void writeBigInteger(Schema schema, BigInteger value) {
+            writeQuery(parentTrait, value.toString());
+        }
+
+        @Override
+        public void writeBigDecimal(Schema schema, BigDecimal value) {
+            writeQuery(parentTrait, value.toString());
+        }
+
+        @Override
+        public void writeString(Schema schema, String value) {
+            writeQuery(parentTrait, value);
+        }
+
+        @Override
+        public void writeBlob(Schema schema, ByteBuffer value) {
+            writeQuery(parentTrait, ByteBufferUtils.base64Encode(value));
+        }
```

Comment:
Let's remove this, blobs are not allowed for query params, see [here](https://smithy.io/2.0/spec/http-bindings.html#httpquery-trait)